values.go

```diff
@@ -38,6 +39,12 @@ func GetFlags(dir string) (map[string]string, error) {
 
 	if version, err := versionFromFile(dir); err != nil {
 		return nil, fmt.Errorf("failed to get version: %v", err)
+	} else if version == "" {
+		// ex: v0.1.2 0.1.2 0.1.2.dev3
+		matched, err := regexp.MatchString(`^v?(\d+\.){1,3}(\d+)`, gitSummary)
```

Comment:
this is still not ok. I'm not gonna accept a version regexp here. This should work on untagged repos as well, therefore please just call repo.Summary() and nothing else here.